frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -747,9 +747,18 @@ class TracingApi {
   }
 
   /**
-   * Returns the length of the shortest path between two nodes in nanometer and in voxels.
-   */
-  measurePathLengthBetweenNodes(sourceNodeId: number, targetNodeId: number): [number, number] {
+   * Returns the length of the shortest path between two nodes in nanometer and in voxels
+   * as well as the shortest path between two nodes as an array of node IDs.
+   * Note that the array is ordered from target to source node.
```

Comment:
the order from target to source is a bit weird. I'd swap it. you can do so by changing `push` to `unshift` at the end of the function when iterating across the parents.